src/third_party/dartino/mark_sweep.h

```diff
@@ -90,83 +90,83 @@ class FreeList {
 #if defined(_MSC_VER)
   // Work around Visual Studo 2013 bug 802058
```

Comment:
Consider removing this, if the bug is old enough.
Also, we currently don't even compile with visual studio code as the interpreter label-jumping is not supported by its compiler.

If you keep it, fix the typo in "Studo".